arviz/data/io_pyjags.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         try:
             import pyjags  # pylint: disable=import-error
             self.pyjags = pyjags
-        except Exception as e:
+        except Exception:
```

Comment:
That's pretty broad. Maybe it would make sense to just catch `ImportError` here?

Reply:
You are right, the linter had complained about this. I changed the exception type to ModuleNotFoundError